doc/api/async_hooks.md

```diff
@@ -973,6 +973,10 @@ chains. It allows storing data throughout the lifetime of a web request
 or any other asynchronous duration. It is similar to thread-local storage
 in other languages.
 
+While you can implement your own implementation on top of the `async_hooks`
+module, `AsyncLocalStorage` is a performant and memory safe implementation that
```

Comment:
Suggestion: change "`AsyncLocalStorage` is a performant and memory safe implementation that involves non-obvious optimizations." to "AsyncLocalStorage` should be preferred over such implementations as it provides a good level of memory safety and benefits from a number of internal optimizations."

Reply:
I find `internal` misleading: as a reader I would interpret it as using internal APIs that are not accessible to me as a user. Which is wrong? I took a quick look at the implementation of `AsyncLocalStorage` and I didn't see any usage of internal APIs (but maybe I've overlooked them).

I just added `should be preferred`.